src/InlineLink/InlineLink.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from "react";
 import "./inline-link-style.scss";
 
-interface InlineLinkProps {
-    onClick?: () => void;
-    className?: string;
+export interface InlineLinkProps {
     children?: any;
```

Comment:
this should be removed. `props.children` exist on every react component already and they are not of type `any`